nuxeo-core/nuxeo-core-event/src/main/java/org/nuxeo/ecm/core/work/StreamWorkManager.java

```diff
@@ -86,6 +91,12 @@
 
     public static final int DEFAULT_CONCURRENCY = 4;
 
+    public static final String KV_NAME = "workManager";
+
+    public static final String STATE = ":state";
+
+    public static final long STATE_TIMEOUT_S = 60 * 60L;
```

Comment:
We should add `@since` java doc declaration.

Reply:
👍